<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speckles</string>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="4">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="0" rowspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1">
       <widget class="QLineEdit" name="sCountEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="pLengthEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="pWidthEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="sDiamEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="wLabel">
        <property name="text">
         <string>Plate width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plate length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="spLabel">
        <property name="text">
         <string>Speckle diameter</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Speckle Count</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Finer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="drPlateButton">
        <property name="text">
         <string>Draw the plate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spButton">
        <property name="text">
         <string>Speckle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanButton">
        <property name="text">
         <string>Clean the plate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clScrButton">
        <property name="text">
         <string>Clean the screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="drPlateWithPaintBtn">
        <property name="text">
         <string>Draw the plate with paintEvent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spButton_2">
        <property name="text">
         <string>Speckle with paintEvent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanButton_2">
        <property name="text">
         <string>Clean the plate with paintEvent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clScrButton_2">
        <property name="text">
         <string>Clean the screen of the QGLWidget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton_2">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="denseSpinBox">
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Coarser</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
